FoodOrderWebApi/Configuration/DbInitializer.cs

```diff
@@ -0,0 +1,215 @@
+﻿using FoodOrderWebApi.Model;
+using NodaTime;
+
+namespace FoodOrderWebApi.Configuration
+{
+    public class DbInitializer
+    {
+        private static FoodOrderDbContext _context = null!;
+
+        public static void Initialize(IServiceProvider serviceProvider)
+        {
+            _context = serviceProvider.GetRequiredService<FoodOrderDbContext>();
+
+            if (!_context.Database.EnsureCreated())
+            {
+                return;
+            }
+
+            SeedCategories();
+            SeedOpeningHours();
+            SeedRestaurants();
+            SeedProducts();
+        }
+        private static void SeedCategories()
+        {
+            var categories = new Category[]
+            {
+                new Category
+                {
+                    Name = "Soup",
+                    PictureName = "soup.jpg"
+                },
+                new Category
+                {
+                    Name = "Drink",
+                    PictureName = "drink.jpg"
+                },
+                new Category
+                {
+                    Name = "Pizza",
+                    PictureName = "pizza.jpg"
+                },
+                new Category
+                {
+                    Name = "Pasta",
+                    PictureName = "pasta.jpg"
+                },
+            };
+
+            foreach (Category c in categories)
+            {
+                _context.Categories.Add(c);
+            }
```

Comment:
Would this Work? 
```suggestion
            _context.Categories.AddRange(categories);
```

[reference](https://learn.microsoft.com/en-us/dotnet/api/system.data.entity.dbset.addrange?view=entity-framework-6.2.0&redirectedfrom=MSDN#overloads)